### ISSUE TITLE
unable to install

### ISSUE THREAD

**Issue by notlebowski** [leonardoxh/race-control-tv#109]
I was made aware by a friend that this app was pulled from the Play Store.

I have a Sony Google TV 2021 and the official app has never been available for download.
Last year I installed this F1 viewer using "Google Play". 
It worked fine last year and it is still working.....   
Many thanks for making this possible

I don't know why F1 TV is now sabotaging this working alternative.
I am a **_paying_** customer ffs.
Just had a long chat-session with F1 TV support, but they were not even aware that their own official app wasn't available on Google TV.

If I use the in-built browser and click on **_app-relase.apk_** I get a message that I can't open the file.

I used the other link for the "test program"
It says "je bent tester" (you are a tester)

1. Verwijder de testversie
2. Installeer de openbare versie via Google Play

Option #2 is clickable, but gives me a special page that say that "the requested URL can't be found on this server"

Is there something wrong? Do I need to wait another day?
What should I expect to happen if I click on that link?

**Comment by notlebowski** [comment]
I tried another method.
downloaded the APK on a stick
then I installed "**TV File commander**"
browsed to the APK
I clicked the APK
I needed to modify the settings, so "TV File commander" was allowed to install from unknown sources.
I clicked the APK again
Now it said it did not install the app....


BTW...  just tried it another time and it also asked me if I wanted to update the one I already have on...

The App is still working, btw
I have no idea if it installed a newer version.
**_Please also add a version somewhere!!!_**

**Comment by notlebowski** [comment]
Strangely enough I was able to install the official F1TV Player with an APK install using the file manager and a USB-stick.

It could well be that this App is supposed to be in the official store before the season really starts.
This could be the reason they asked this App to be taken down from the store, so it will not confuse the ones that install it.

I have now both players on my Google TV.
I still don't know why the apk of THIS app refused to install through the file manager whereas the official one did.
Maybe the official app has already a special signature for Google TV _(speculating here, I don't know if such a thing even exists)_  ???

**Comment by notlebowski** [comment]
Thanks Leonardo,

I was fully aware your App has been taken down. I fully blame F1TV for that.
I'm currently writing back and forth to F1TV-support and am complaining about that action.

Like I wrote in the 2nd post. I was able to sideload the official app of March and it is working fine. I now have both apps on my Google TV.

For the time being I don't want to risk losing your app on my TV, so I will not uninstall the app yet. I will ask a friend of mine who just bought the F1 subscription and doesn't have your app on it.

If he succeeds, I will try it again.

**Comment by notlebowski** [comment]
These are the latest responses from "F1 TV"

Thanks for contacting Formula 1. Your reference number for this case is 00356452.



```
This is not the experience we want for our fans. We sincerely apologize for the inconvenience this has caused you. Rest assured that the reason for this is due to updates that are on going. We are still currently working on a fix.



You can also check out our FAQ's (https://support.f1.tv/s/?language=en_US<https://support.f1.tv/s/?language=en_US>) where you may be able to find additional information to help you.
```

```
Thank you for your response.

I hope there were actually no grounds for my fears.

To summarize:

Like I wrote. Both Android Apps work on my Google TV.
The only problem is that I can't install those using Google Play.

This is not a technical issue, but a decision from your part.

You decided to request a takedown from the alternate App on Google Play by writing leonordaxh
You decided to exclude TV's from Google Play for your official Android App.

I hope you will now decide to revert one of those decisions before the season starts.
```

**Comment by leonardoxh** [comment]
Well, technically we are indeed infringing not copyrights but also some user terms because we are doing reverse engineering on their apis.

I've seen some other apps being removed as well, which is, well a pity, but I guess that is it, really hope their app can catchup, because last time I used it, it was very laggy on my Chromecast with Google TV

**Comment by notlebowski** [comment]
My install of the F1-app failed because I had the old App still on my Google-TV (the one I once installed using Google Play).
Because I didn't want to remove the app for being afraid my install would fail anyhow, I wasn't sure this was the case.

Because my old App stopped working because the login failed due to a change at F1, I tried to install the newest version.
I removed the old App and installed the new one using the USB-stick.
This worked!
Just had to login again!!

Great work and keep it up!
I'm still using the official App I downloaded from APKmirror, but I really want to have an alternate app if it somehow deserts me.
